test/Confluent.Kafka.IntegrationTests/Tests/Consumer_OperationalScenarios.cs

```diff
@@ -0,0 +1,140 @@
+// Copyright 2016-2017 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using Confluent.Kafka.Serialization;
+
+
+namespace Confluent.Kafka.IntegrationTests
+{
+    public static partial class Tests
+    {
+        /// <summary>
+        ///     Tests various
+        /// </summary>
+        [Theory, MemberData(nameof(KafkaParameters))]
+        public static void Consumer_OperationalScenarios(string bootstrapServers, string topic)
+        {
+            var consumerConfig = new Dictionary<string, object>
+            {
+                { "group.id", "simple-produce-consume" },
+                { "bootstrap.servers", bootstrapServers },
+                { "session.timeout.ms", 6000 }
+            };
+
+            // VALID Scenarios
+
+            // start then stop then start a Consumer
```

Comment:
May I suggest splitting these blocks to separate tests with distinct names? IMO it would greatly improve readability of the tests. Also as you will get multiple test failures instead of just one, it may help to find out the root cause faster.